concepticondata/concepticon.tsv

```diff
@@ -572,21 +572,21 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 571	STING	The body	To hurt, usually by introducing poison or a sharp point, or both.	Action/Process	
 572	STREW	Motion	To distribute objects or pieces of something over an area, especially in a random manner.	Action/Process	
 573	SWAY	Motion	To move or swing from side to side; or backward and forward; to rock.	Action/Process	
-574	TRANSIT	Motion	To pass over, across or through something.	Action/Process	
+574	TRANSIT	Motion	To cross something (a river, a street etc.).	Action/Process	
```

Comment:
This is based on the actual words mapped to this concept? TRANSIT as a gloss seems more general and less specific, almost MOVE (or perhaps the idea is to differentiate from MOVE?)

Reply:
Transit includes elicitations like "cross (river)", whereas traverse is "go through (a hole)".

Reply:
We might want to change the name of them of course, but I tried to avoid that.

Reply:
Making glosses more transparent is the best choice.

Reply:
Should this be done by adding a new concept and moving all links to the new concept, or by renaming the concept directly, changing only the name and not the number in the concept lists? (so far I only used the first method except for in one case, as far as I see).

Reply:
I'd change the gloss: we should be really careful about adding new concepts, especially when there is even a partial overlapping, as Concepticon is not intended to expand to include all possible meanings and concepts (imagine if we were to add even conlangs, like Klingon).